### ISSUE TITLE
Wordpress LetsEncrypt Doc seems outdated

### ISSUE THREAD

**Issue by wsoyka** [awsdocs/amazon-lightsail-developer-guide#56]
I've just tried to follow the doc [amazon-lightsail-using-lets-encrypt-certificates-with-wordpress.md](../blob/master/doc_source/amazon-lightsail-using-lets-encrypt-certificates-with-wordpress.md), for a new bitnami wordpress (debian) instance - but at the end the Really Simple SSL Pugin failed to find the certs. 

On closer investigation, it seems like the locations for htaccess.conf and wp-config.php are outdated in the doc. 

I have not been able to figure out what paths I'd need to use for it to work - but I then checked out [the Bitnamis doc for SSL on AWS](https://docs.bitnami.com/aws/how-to/generate-install-lets-encrypt-ssl/) which not only worked, but provide a much more elegant solution - a script that helps you set SSL up alot easier than outlined in the docs here. It also seems like this script provides automatic renewal via a cronjob.

Possible solutions: Either update the doc to the correct paths or link to the Bitnami docs?

**Comment by wsoyka** [comment]
I just noticed the doc mentions the bncert tool and that you already have a doc for its use. The paths in the mentioned doc are still wrong though as far as I can tell.

**Comment by AvverbioPronome** [comment]
They're not outdated. They just don't make any sense. It does not make any sense to make the user install certbot when bitnami images already provide an acme client and they actually wrote docs on using it. 

That tutorial is doing a lot of damage. Just remove it and point people to bncert or the docs on bitnami's website.

**Comment by rlatino** [comment]
We are considering unpublishing the Let's Encrypt tutorials. In the meantime, we have updated the directory paths in the tutorial to the correct ones. I've tested it, and they work.

**Comment by AvverbioPronome** [comment]
> I've tested it, and they work.

Yeah, probably. The issue is mainly sending inexperienced people to use the **manual** dns-01 challenge without them realizing they need to repeat the operation every 60-90 days. 

Because they usually don't need a wildcard certificate, there is no reason for them to use the dns-01 challenge, and they would be *objectively* better served by sticking to the solution bitnami provides in their images, bncert, which autorenews and unless something really bad happens requires no user intervention after initial setup.

**Comment by Ger-JanF** [comment]
> We are considering unpublishing the Let's Encrypt tutorials.

I'm not sure if you're refering to *all* Let's Encrypt tutorials, but there are a few that are indeed damaging to the Let's Encrypt "experience". Any *manual* step in *any* tutorial needs to be removed, as Let's Encrypt is intended to be automated.

**Comment by osirisinferi** [comment]
> We are considering unpublishing the Let's Encrypt tutorials.

I'm not sure if you're refering to all Let's Encrypt tutorials, but there are a few that are indeed damaging to the Let's Encrypt "experience". Any manual step in any tutorial needs to be removed, as Let's Encrypt is intended to be automated.
